Adyen Klarna Doc Update (#5927)
vuestorefront/vue-storefront: packages/core/docs/integrations/adyen.md

  * `ctApi` - You need `manage_orders` and `manage_payments` scopes to make it work properly, base on [that page](../commercetools/getting-started.html#configuring-your-commercetools-integration) during configuring this property. Then for `apiHost` you have to use only the base URL - `https://<SHOP_DOMAIN>.com/` instead of `https://<SHOP_DOMAIN>.com/vsf-ct-dev/graphql`
  * `adyenMerchantAccount` - Name of your Adyen's merchant account
  * `origin` - URL of your frontend. You could check it by printing out `window.location.origin` in the browser's console on your website.
  * `buildRedirectUrlAfterAuth` - `(paymentAndOrder: PaymentAndOrder, succeed: boolean) => string` - A method that tells the server where to redirect the user after coming back from payment gateway. You can test it with [these cards](https://docs.adyen.com/development-resources/test-cards/test-card-numbers#test-3d-secure-authentication).

>   * `buildRedirectUrlAfter3ds1Auth` - deprecated `buildRedirectUrlAfterAuth`

Reviewer: ```suggestion
  * `buildRedirectUrlAfter3ds1Auth` - deprecated in favor of `buildRedirectUrlAfterAuth`
```

  * `buildRedirectUrlAfterError` - `(err: Error) => string` - A method that tells the server where to redirect the user if error has been thrown inside `cardAuthAfterRedirect` controller.

>   * `buildRedirectUrlAfter3ds1Error` - deprecated `buildRedirectUrlAfterError`

Reviewer: ```suggestion
  * `buildRedirectUrlAfter3ds1Error` - deprecated in favor of `buildRedirectUrlAfterError`
```

### Paypal configuration
Configuration of PayPal is well-described in [Adyen's documentation](https://docs.adyen.com/payment-methods/paypal/web-drop-in).

### Klarna configuration
To enable Klarna, you have to add a new payment method in Adyen's dashboard. Then you should add specified methods to the `availablePaymentMethods` array in `nuxt.config.js`:
```js
// nuxt.config.js

export default {
  modules: [
    ['@vsf-enterprise/adyen/nuxt', {
      availablePaymentMethods: [
        'scheme',
        'paypal',
        'klarna',
        'klarna_account',
        'klarna_paynow'
      ],
      // ...
    }]
  ]
};
```

> Read [Adyen's document aboout the Klarna](https://docs.adyen.com/payment-methods/klarna#supported-countries) to check which Klarna payment method is available for which country.

Reviewer: ```suggestion
Read [Adyen's document about the Klarna](https://docs.adyen.com/payment-methods/klarna#supported-countries) to check which Klarna payment methods are available for individual countries.
```

:::warning Phone number

> If your users may provide phone number then make sure it is with **area code**. Otherwise, Klarna will throw an error because of unproper phone number.

Reviewer: ```suggestion
If your users can provide a phone number then make sure it is with **area code**. Otherwise, Klarna will throw an error because of an improper phone number format.
```

- [Notification Module](https://github.com/commercetools/commercetools-adyen-integration/blob/master/notification/docs/HowToRun.md#commercetools-project-requirements).   

For more information, see the 6th step of the [Adyen's installation guide](./adyen.html#installation).

### Klarna Pay Later does not work for US

> Klarna Pay Later is not supported in US. However, sometimes when you add Klarna in the Adyen's dashboard - it is accidently added too. If you have this problem, contact the support so they will remove this one for the US. 

Reviewer: ```suggestion
Klarna Pay Later is not supported in the United States. However, sometimes it is added when you enable Klarna in Adyen's dashboard. If you have this problem, contact Adyen's support to remove it.
```
